sklearn/metrics/_classification.py

```diff
@@ -1463,6 +1468,17 @@ def fbeta_score(
     >>> fbeta_score(y_true, y_pred_empty,
     ...             average="macro", zero_division=np.nan, beta=0.5)
     0.12...
+
+    In order to use :func:`fbeta_scorer` as a scorer, a callable
+    scorer objects needs to be created first with :func:`make_scorer`,
+    passing the value for the `beta` parameter.
+
+    >>> from sklearn.metrics import fbeta_score, make_scorer
+    >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
+    >>> from sklearn.model_selection import GridSearchCV
+    >>> from sklearn.svm import LinearSVC
+    >>> grid = GridSearchCV(LinearSVC(dual="auto"), param_grid={'C': [1, 10]},
+    ...                     scoring=ftwo_scorer, cv=5)
```

Comment:
Let's follow the black formatting conventions in our docstrings as well (even if it was not always the case in the past.

```suggestion
    >>> grid = GridSearchCV(
    ...     LinearSVC(dual="auto"),
    ...     param_grid={'C': [1, 10]},
    ...     scoring=ftwo_scorer,
    ...     cv=5,
    ... )
```

Reply:
We might want to update the multiline call to `fbeta_score` in the previous example snippet to follow the same format style.